ipatests/azure/scripts/setup_containers.py

```diff
@@ -195,7 +195,7 @@ def setup_container_overrides(self):
         """
         Set services known to not work in containers to be ignored
         """
-        for service in ['nis-domainname',]:
+        for service in ['nis-domainname', 'chronyd']:
```

Comment:
Sorry, where can I see the error logs without masking chronyd?

Reply:
Sadly, the build was already removed by Azure -- I only have references left in the red 'x' sign next to 'spec: use pkgconf ..' commit before I re-pushed the masking of chronyd.

![Screenshot from 2020-11-17 17-53-10](https://user-images.githubusercontent.com/2066/99412800-ca409880-28fd-11eb-915a-8ad523f47618.png)

It was also visible in the rawhide runs. I suspect it is a sideeffect of seccomp update?